Twilio/TwiML/Fax/Receive.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * This code was generated by
+ * \ / _    _  _|   _  _
+ * | (_)\/(_)(_|\/| |(/_  v1.0.0
+ * /       /
+ */
+
+namespace Twilio\TwiML\Fax;
+
+use Twilio\TwiML\TwiML;
+
+class Receive extends TwiML {
+    /**
+     * Receive constructor.
+     * 
+     * @param array $attributes Optional attributes
+     */
+    public function __construct($attributes = array()) {
+        parent::__construct('Receive', $attributes);
+    }
+
+    /**
+     * Add Action attribute.
+     * 
+     * @param url $action Receive action URL
+     * @return TwiML $this.
+     */
+    public function setAction($action) {
+        return $this->setAttribute('action', $action);
+    }
+
+    /**
+     * Add Method attribute.
+     * 
+     * @param http_method $method Receive action URL method
```

Comment:
nit: is there any built in support for having things like this show up as camelCase to match surrounding code?

Reply:
sure!